### PR TITLE
Fix pig latin challenge description

### PR DESCRIPTION
**Pull request by crandalj**
Added statement for beginning with consonants as it is a bit unclear if you just read the two existing statements.

- [x ] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x ] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x ] My pull request targets the `master` branch of freeCodeCamp.
- [x ] None of my changes are plagiarized from another source without proper attribution.
- [x ] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x ] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/pig-latin.english.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ forumTopicId: 16039
 Translate the provided string to pig latin.
 <a href="http://en.wikipedia.org/wiki/Pig_Latin" target="_blank">Pig Latin</a> takes the first consonant (or consonant cluster) of an English word, moves it to the end of the word and suffixes an "ay".
 If a word begins with a vowel you just add "way" to the end.
+If a word begins with consonants, move them to the end followed by "ay".
```
Comment (RandellDawson) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/pig-latin.english.md]: The next line already explains what to do if the word does not contain a vowel.  It says to `just add "ay" to the end`.

Also, your addition is not accurate, because just because the word starts with consonants, does not mean it does not have vowels after those consonants.

Comment (crandalj) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/pig-latin.english.md]: It says if it contains no vowels, which is very specific. The words you convert contain vowels, and my statement is after it already tells you to move consonants to the end of the word, however that bit is tucked away and not prominent like the other two statements.

Comment (RandellDawson) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/pig-latin.english.md]: The definition of Pig Latin already explains what should happen with the first consonant or consonant cluster:
> Pig Latin</a> takes the first consonant (or consonant cluster) of an English word, moves it to the end of the word and suffixes an "ay".

Comment (crandalj) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/pig-latin.english.md]: If it explained it well I wouldn't be here with a suggestion. On the description it clearly displays the rules separate from the first sentence. I do not see why you are against clarifying a prompt to avoid confusion.

I find it pointless to argue about opinion on how to display something, I'll let whomever has the ability to review it decide, however I did waste time trying to complete this incorrectly because of how it is presented.

**Comment by RandellDawson** [comment]
> I'll let whomever has the ability to review it decide, 

I am reviewing it.  I personally do not think this change is needed, but I will let other mods voice their opinion also.  Together we will come to a consensus of how to proceed with this PR.

Thank you for your input.

**Comment by ojeytonwilliams** [comment]
I redid the challenge and I don't think it's terribly worded.  We could perhaps make it clearer, though.  How about:

- If a word begins with a vowel just add "way" to the end.
- Otherwise take the first consonant (or consonant cluster), move it to the end of the word and append "ay"
- If there are no vowels, you should still append "ay"

The third rule *is* technically redundant (it's covered by the second rule), but it should make it clear that you should do something, even if there are no vowels.

**Review by Manish-Giri — COMMENTED** [review]
I don't see what the proposed change - 

> If a word begins with consonants, move them to the end followed by "ay"

adds to the challenge that isn't already covered - 

> Pig Latin takes the first consonant (or consonant cluster) of an English word, moves it to the end of the word and suffixes an "ay".

The test cases also cover all possible scenarios. 

---

@crandalj Since you mentioned you solved the challenge incorrectly because of how the challenge is presented, perhaps you could specify what your confusion was? If you are referring to - 
> Pig Latin takes the first consonant (or consonant cluster) of an English word, moves it to the end of the word and suffixes an "ay".

not being in the form of a possible test case like the other two instructions, `"if a word ..."`, we could perhaps reword it to make it seem like an instruction, but adding what you proposed, **in addition** to what is already there, is redundant. 

If a change is necessary, how about something on the lines of -

> Translate the provided string to [Pig Latin](URL).

> Take the first consonant (or consonant cluster) of an English word, move it to the end of the word and suffixes an "ay".

> If a word begins with a vowel, you just add "way" to the end.

> If a word does not contain a vowel, just add "ay" to the end.

**Comment by crandalj** [comment]
@Manish, just close it then. I think the wording on a lot of challenges needs reviewed but this one in particular is the one that annoyed me. Since no one can see the flaw in how it’s structured then I guess I am alone in this.

**Comment by moT01** [comment]
This PR has been sitting here for a bit. I kind of agree that it doesn't quite seem right. The problem is that the first line is referring to "Pig Latin", while the second two lines are referring to "you" as in the user. So the first line doesn't feel like an instruction. What about this...

```
Pig Latin is a way of altering English Words. The rules are as follows:

If a word begins with a consonant, take the first consonant or consonant cluster, move it to the end of the word, and add "ay" to it.

If a word begins with a vowel, just add "way" at the end.

Translate the provided string to Pig Latin. Input strings are guaranteed to be English words in all lowercase.
```

I also don't mind the other two suggestions, but I think I like mine.

**Comment by RandellDawson** [comment]
I like Tom's suggestion.  It gets rid of the redundant "If a word does not contain a vowel, just add "ay" to the end. " and clearly separates the description from the instruction.  I would also suggest moving the instruction part within the `instructions` sections to make it more clear.

@ojeytonwilliams @Manish-Giri If you are ok with that, then let us know, so we can instruct the OP to make the applicable changes and merge something that makes any improvement to what is currently written.  Otherwise, we should just close this PR.

**Comment by ojeytonwilliams** [comment]
Sure, let's go with Tom's suggestion.

**Comment by Manish-Giri** [comment]
I'm also fine with the suggestion, let's go ahead.

**Comment by RandellDawson** [comment]
@crandalj Can you please implement @moT01's suggestion?  Thank you.

**Comment by RandellDawson** [comment]
@crandalj @ojeytonwilliams @moT01 @Manish-Giri In an effort to save time, I have made the suggestions made by @moT01.  I added an undorded list to the description, but I am fine changing it to something else if need be.  Also, I moved the instructions to the `Instructions` section.

Can we get two final reviews on the changes?

Thanks.

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/pig-latin.english.md [hunk 2]
```diff
@@ -8,16 +8,16 @@ forumTopicId: 16039
...
 ## Description
 <section id='description'>
-Translate the provided string to pig latin.
-<a href="http://en.wikipedia.org/wiki/Pig_Latin" target="_blank">Pig Latin</a> takes the first consonant (or consonant cluster) of an English word, moves it to the end of the word and suffixes an "ay".
-If a word begins with a vowel you just add "way" to the end.
-If a word does not contain a vowel, just add "ay" to the end.
-Input strings are guaranteed to be English words in all lowercase.
+Pig Latin is a way of altering English Words. The rules are as follows:
+- If a word begins with a consonant, take the first consonant or consonant cluster, 
+move it to the end of the word, and add "ay" to it.
```
Comment (ojeytonwilliams) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/pig-latin.english.md]:
```suggestion
- If a word begins with a consonant, take the first consonant or consonant cluster, move it to the end of the word, and add "ay" to it.
```
The linebreak was creating a newline

Comment (moT01) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/pig-latin.english.md]:
```suggestion
- If a word begins with a consonant, take the first consonant or consonant cluster, move it to the end of the word, and add "ay" to it.
```

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/pig-latin.english.md [hunk 3]
```diff
@@ -8,16 +8,16 @@ forumTopicId: 16039
...
+- If a word begins with a vowel, just add "way" at the end.
 </section>
 
 ## Instructions
 <section id='instructions'>
-
+Translate the provided string to Pig Latin. Input strings are guaranteed to be 
+English words in all lowercase.
```
Comment (moT01) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/pig-latin.english.md]:
```suggestion
Translate the provided string to Pig Latin. Input strings are guaranteed to be English words in all lowercase.
```

Comment (ojeytonwilliams) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/pig-latin.english.md]:
```suggestion
Translate the provided string to Pig Latin. Input strings are guaranteed to be English words in all lowercase.
```

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]

**Review by moT01 — CHANGES_REQUESTED** [review]
~I put line breaks in my comment so it looked better on github. You must have copy/pasted that. It doesn't looks good on site with them.~ 
Edit: looks like Oliver beat me to it.

I don't mind the unordered list, but wondering if an ordered list would be better here. Either way works for me.

**Comment by RandellDawson** [comment]
> I don't mind the unordered list, but wondering if an ordered list would be better here. Either way works for me.

I don't really think an ordered list is needed as there is not a specific order in which the requirements must be performed.  I think of an ordered list as steps in a process where order matters.

**Review by moT01 — APPROVED** [review]
LGTM 🎉

**Review by ojeytonwilliams — APPROVED** [review]
LGTM 👍
